app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+class Api::V1::CommentsController < Api::V1::BaseController
+  before_action :authenticate_user!, except: %i[index]
```

Comment:
What about comment of private projevts/circuits

Reply:
Can `private_projects` have comments? I preassumed them to not have one.. @tachyons

Reply:
Can you double check ?

Reply:
Sure @tachyons

Reply:
No distinction for comments in regard to `project_access_type`, will add required checks soon!!